core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -149,7 +149,7 @@ class Controller(
   implicit val ec: ExecutionContext = context.dispatcher
   implicit val timeout: Timeout = 5.seconds
 
-  lazy val promiseHandlerInitializer = wire[PromiseHandlerInitializer]
+  lazy val rpcHandlerInitializer = wire[RPCHandlerInitializer]
```

Comment:
why do we need `rpcHandlerInitializer ` here again? It's already in WorkflowActor

Reply:
WorkflowActor's rpcHandlerInitializer is abstract.